app/views/speakers/_form.html.erb

```diff
@@ -11,71 +11,72 @@
     </div>
   <% end %>
 
-  <h2>Speaker</h2>
-
-  <div class="field">
-    <%= form.label :name %>
-    <%= form.text_field :name, id: :speaker_name %>
-  </div>
-
-  <div class="field">
-    <%= form.label :picture %>
-    <%= form.file_field :picture %>
-  </div>
-
-  <div class="field">
-    <%= form.label :email %>
-    <%= form.text_field :email %>
-  </div>
-
-  <div class="field">
-    <%= form.label :company %>
-    <%= form.text_field :company %>
-  </div>
-
-  <div class="field">
-    <%= form.label :biography %>
-    <%= form.text_area :biography %>
-  </div>
+  <div class="form">
+    <h2>Palestrante</h2>
```

Comment:
Its better to leave the `h2` outside the form, since it is not a content of the form.